backend/src/controllers/auth.py

```diff
@@ -19,9 +19,10 @@ def login():
     @auth_controller.route("get-user-code")
     def auth_redirect():
         code = request.args.get("code")
-        state = request.args.get("state")
-        if state != session["SpotifyState"]:
-            return redirect("/#" + urlencode({"error": "state_mismatch"}))
+        # state = request.args.get("state")
+        # test = session["SpotifyState"]
```

Comment:
Seeming to have trouble persisting the session state in the Azure deployment. Not sure if individual requests from the same browser are being treated as random by Azure. Could imagine so for scalability reasons?